docs/glossary.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Glossary
+sidebar_label: Glossary
+keywords:
+- Harvester
+- glossary
+- terminology
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/glossary"/>
+</head>
+
+## **guest cluster** / **guest Kubernetes cluster**
+
+Group of integrated Kubernetes worker machines that are running in a VM that is part of a Harvester cluster. Guest clusters form the underlying infrastructure for running containerized workloads.
```

Comment:
Formulation a bit unclear. Is the guest cluster _part_ of the Harvester cluster, or is it a separate cluster of VM workload _on top_ of the Harvester cluster?
Is a guest cluster required for containerized workloads or can container workloads not  also be deployed directly on the Harvester cluster?
Also is this a group of K8s workers in a single VM, or a group of VMs, each being a K8s worker node?

I thought the situation with a guest cluster was like this:
```
  ┌─────────────────────────────────────────────────────────────────────────┐
  │ Harvester Cluster                                                       │
  │ ┌───────────────────┐ ┌───────────────────┐ ┌───────────────────┐       │
  │ │Physical Server    │ │Physical Server    │ │Physical Server    │       │
  │ │                   │ │                   │ │                   │       │
  │ │Harvester Node 1   │ │Harvester Node 2   │ │Harvester Node 3   │       │
  │ ├───────────────────┤ ├───────────────────┤ ├───────────────────┤       │
  │ │                   │ │                   │ │                   │       │
  │ │                   │ │                   │ │                   │       │
  │ │ ┌─────────────────┼─┼───────────────────┼─┼───────────────────┼─────┐ │
  │ │ │ Guest Cluster   │ │                   │ │                   │     │ │
  │ │ │ ┌───────────┐   │ │   ┌───────────┐   │ │   ┌───────────┐   │     │ │
  │ │ │ │VM         │   │ │   │VM         │   │ │   │VM         │   │     │ │
  │ │ │ │           │   │ │   │           │   │ │   │           │   │     │ │
  │ │ │ │K8s Node 1 │   │ │   │K8s Node 2 │   │ │   │K8s Node 3 │   │     │ │
  │ │ │ ├───────────┤   │ │   ├───────────┤   │ │   ├───────────┤   │ ... │ │
  │ │ │ │Containers/│   │ │   │Containers/│   │ │   │Containers/│   │     │ │
  │ │ │ │Pods       │   │ │   │Pods       │   │ │   │Pods       │   │     │ │
  │ │ │ │           │   │ │   │           │   │ │   │           │   │     │ │
  │ │ │ │           │   │ │   │           │   │ │   │           │   │     │ │
  │ │ │ └───────────┘   │ │   └───────────┘   │ │   └───────────┘   │     │ │
  │ │ │                 │ │                   │ │                   │     │ │
  │ │ └─────────────────┼─┼───────────────────┼─┼───────────────────┼─────┘ │
  │ │                   │ │                   │ │                   │       │
  │ └───────────────────┘ └───────────────────┘ └───────────────────┘       │
  │                                                                         │
  └─────────────────────────────────────────────────────────────────────────┘
```

Reply:
> Formulation a bit unclear. Is the guest cluster _part_ of the Harvester cluster, or is it a separate cluster of VM workload _on top_ of the Harvester cluster?

The latter. I'd suggest wording it something like: ...machines that are running in VMs on top of a Harvester cluster" (note: "VMs" plural, and "on top of" as you suggested).

> Is a guest cluster required for containerized workloads or can container workloads not also be deployed directly on the Harvester cluster?

Deploying container workloads directly on the Harvester cluster is possible, but is still experimental AIUI (see https://docs.harvesterhci.io/v1.3/rancher/rancher-integration#harvester-baremetal-container-workload-support-experimental)

> Also is this a group of K8s workers in a single VM, or a group of VMs, each being a K8s worker node?

Should be a group of VMs - I believe you diagram is correct.

Reply:
@m-ildefons @tserong Thank you for pointing out the deficiencies of the draft definition. Please check the updated one.